.github/workflows/version.yaml

```diff
@@ -0,0 +1,141 @@
+name: Bump Version and Generate Docs
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
maybe filter by PRs targetting `main` or repo's default branch?

Also, dunno how I feel running this on the PR vs on commits to `main` - IMO the latter would prevent any churn on the PR branch and might reduce any risks with commits